restore/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Restore Only Cache'
+description: 'Restore Cache artifacts like dependencies and build outputs to improve workflow execution time'
+author: 'GitHub'
+inputs:
+  path:
+    description: 'The same list of files, directories, and wildcard patterns to restore cache that were used while saving it'
+    required: true
+  key:
+    description: 'An explicit key for restoring the cache'
+    required: true
+  restore-keys:
+    description: 'An ordered list of keys to use for restoring stale cache if no cache hit occurred for key. Note `cache-hit` returns false in this case.'
+    required: false
+outputs:
+  cache-hit:
+    description: 'A boolean value to indicate an exact match was found for the primary key'
+  cache-primary-key:
+    description: 'Cache primary key passed in the input to use in subsequent steps of the workflow'
+  cache-restore-key:
+    description: 'Cache key restored'
```

Comment:
```suggestion
    description: 'Restore key which was used to restore the cache. It will not be set in case there was an exact match with primary key itself'
```